redfish-core/lib/power_subsystem.hpp

```diff
@@ -4,9 +4,122 @@
 #include <utils/chassis_utils.hpp>
 #include <utils/json_utils.hpp>
 
+#include <cstdint>
+
 namespace redfish
 {
 
+// Map of service name to list of interfaces
+using MapperServiceMap =
```

Comment:
upstream we have moved to a common type: e.g. https://github.com/openbmc/bmcweb/commit/b9d36b4791d77a47e1f3c5c4564fcdf7cc68c115

Reply:
Done